tensorflow_datasets/text/BablTasks.py

```diff
@@ -0,0 +1,165 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import re
+import os
+from absl import logging
+import tensorflow as tf
+from tensorflow_datasets.core import api_utils
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """\
+@article{DBLP:journals/corr/WestonBCM15,
+  author    = {Jason Weston and
+               Antoine Bordes and
+               Sumit Chopra and
+               Tomas Mikolov},
+  title     = {Towards AI-Complete Question Answering: {A} Set of Prerequisite Toy
+               Tasks},
+  journal   = {CoRR},
+  volume    = {abs/1502.05698},
+  year      = {2015},
+  url       = {http://arxiv.org/abs/1502.05698},
+  archivePrefix = {arXiv},
+  eprint    = {1502.05698},
+  timestamp = {Mon, 13 Aug 2018 16:47:58 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/WestonBCM15},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+"""
+_DESCRIPTION = """\
+This dataset is the collection of 1000 training and test data which deals with the collection of certain \
+text with every 3rd line containing some questions based on the previous values giving certain answers \
+related to the dataset and with the id number related to the answer
+"""
+
+_DOWNLOAD_URL = ("http://www.thespermwhale.com/jaseweston/babi/"
+                 "tasks_1-20_v1-2.tar.gz")
+_TOP_LEVEL_DIR = "tasks_1-29_v1-2"
+
+_TRAIN_FILE_FORMAT = os.path.join(_TOP_LEVEL_DIR, "shuffled-10k", "qa(.*)train.txt")
```

Comment:
This is a glob, not regexp.

Try changing to "qa*train.txt"

Reply:
ohk sir @cyfra  thanks for your help!!
Can you reassign me this issue again?

Reply:
Same error still persists changed the code accordingly!!!

Reply:
did you also change the qa(.*)test in the line below ?

Can you push the freshest version of the code ?

Reply:
ok sir sure